easybuild/easyconfigs/t/TopHat/TopHat-2.0.14-goolf-1.7.20.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.0.14'
+
+homepage = 'http://ccb.jhu.edu/software/tophat/'
+description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = ['tophat-%(version)s-zlib.patch']
```

Comment:
if this is a copy of the existing patch (looks like it, since it mentions 2.0.13), just use the existing patch instead please, we have enough patches to take care of already, no need to duplicate them